src/main/java/org/cardanofoundation/job/repository/ledgersyncagg/AddressTxAmountRepository.java

```diff
@@ -24,11 +24,11 @@ public interface AddressTxAmountRepository
   @Query(
       value =
           """
-      select (case when ata.stake_address is null then ata.address else ata.stake_address end) as account, count(distinct(ata.tx_hash)) as txCount
-      from address_tx_amount ata
-      where ata.epoch = :epochNo
-      and ata.slot != -1
-      group by account
+          select (case when ata.stake_address is null then ata.address else ata.stake_address end) as account, count(distinct(ata.tx_hash)) as txCount
```

Comment:
For code quality I prefer writing SQL Statements in upper case so that it's easier to read. 
And we are doing it in every other Query so just for continuity it is good to have the same style as well.

Reply:
Updated!